coverage.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" ?>
```

Comment:
Argh! That's not even supposed to be there. I will add it to the gitignore file and remove the file from the repo.